example-unsigned/plugin.yaml

```diff
@@ -1,7 +1,6 @@
 apiVersion: v1
 name: example-unsigned
 version: 0.1.0
-config: {}
```

Comment:
config can be (I made it) optional

Reply:
i agree optional is good. was it not before? anyway, lgtm